### PR TITLE
https://github.com/open-cluster-management/backlog/issues/1664

### PR DESCRIPTION
**Pull request by bcpswope**
Dev review

### CODE REVIEW

File: manage_applications/managing_subscriptions.md [hunk 1]
```diff
@@ -391,7 +391,7 @@ spec:
       - start: "10:20AM"
         end: "10:30AM"
       - start: "12:40PM"
-        end: "1:40PM"
+        end: "1:40PM" <!--Ian please see this format--user test threw an error -->
 ```
```
Comment (bcpswope) [inline manage_applications/managing_subscriptions.md:395]: @ianzhang366 to test here

File: manage_applications/managing_subscriptions.md [hunk 2]
```diff
@@ -500,7 +500,7 @@ You can use `spec.packageOverrides` to override `kustomization` at the subscript
 apiVersion: apps.open-cluster-management.io/v1
 kind: Subscription
 metadata:
-  name: example-subscription
+  name: example-subscription <!-- Ian need to talk to you about this -->
   namespace: default
```
Comment (bcpswope) [inline manage_applications/managing_subscriptions.md:504]: @rokej needed here @ianzhang366

Comment (rokej) [reply manage_applications/managing_subscriptions.md:504]: Replace the whole YAML with this.

```
apiVersion: apps.open-cluster-management.io/v1
kind: Subscription
metadata:
  name: example-subscription
  namespace: default
spec:
  channel: some/channel
  packageOverrides:
    - packageName: kustomization
      packageOverrides:
      - value: | 
patchesStrategicMerge:
- patch.yaml
namePrefix: demo-
```

Comment (bcpswope) [reply manage_applications/managing_subscriptions.md]: Done. @ianzhang366 all that is left on this PR is a check on the timewindow format.

File: manage_applications/managing_subscriptions.md (hunk 1 above)
Comment (ianzhang366) [reply manage_applications/managing_subscriptions.md:395]: Hi @bcpswope ,

I tested this on my env, I don't see any error tho...

Here's the trace,
```
I0424 09:17:40.280562   98254 timewindow.go:81] Time window checking at 2020-04-24 15:17:40.280028 +0200 CEST
I0424 09:17:40.280757   98254 timewindow.go:130] start time paresed as 5:00PM, end time 10:00PM
I0424 09:17:40.280776   98254 timewindow.go:130] start time paresed as 12:40PM, end time 1:40PM
I0424 09:17:40.280859   98254 subscriber_item.go:94] Subcription bookinfo/bookinfo-github will de deploy after 1h43m0s
```

Here's the subscription's time window, I used for testing,
```
  timewindow:
    daysofweek:
    - Friday
    - Saturday
    - Sunday
    hours:
    - end: 10:00PM
      start: 05:00PM
    - end: 1:40PM
      start: 12:40PM
    location: Europe/Madrid
    windowtype: active
```

**Pull request merged**